tools/protodoc/templates/extension.rst.tpl

```diff
@@ -15,7 +15,7 @@ This extension has the qualified name ``{{extension}}``
 {% endfor %}
 
 {% if type_urls|length > 0 %}
-  This extension can be configured with the following type URLs:
+  This extension must be configured with the following type URLs:
```

Comment:
```suggestion
  This extension must be configured with one of the following type URLs:
```

not sure about this suggestion, and the pluralization is a bit wierd - but i think this could mean must be configured with all type urls

Reply:
Yes, updated once again. It's just one URL 90% cases, so we can maybe add two types of templates, for single one and plural separately.